scripts/build-benchmarks

```diff
@@ -3,5 +3,8 @@
 # generate benchmarks
 cat data/historical-benchmarks/2015.csv | node scripts/parse-benchmarks-data.js 2015 2017
 
+# merge the benchmark files
+echo "$(node scripts/merge-benchmarks.js ../benchmarks/2017.json ../data/wi_2017.json)" > benchmarks/2017.json
```

Comment:
we can use [`jq`](https://stedolan.github.io/jq/) in lieu of the `merge-benchmarks` script below - jq is super handy when working with json:

```bash
echo "$(jq -s add benchmarks/2017.json data/wi_2017.json)" > benchmarks/2017.json
```